spec/system/home_page_spec.rb

```diff
@@ -1,33 +0,0 @@
-require "rails_helper"
-
-RSpec.describe "Home Page", type: :system do
-  scenario "User visits the claims homepage" do
-    given_i_am_on_the_claims_site
```

Comment:
You've introduced `service:` metadata as a new way to specify which service the test runs against. So should we remove these helper methods now?

https://github.com/DFE-Digital/itt-mentor-services/blob/a691cfdb0ba7ff8367606f4598a855ee8deb9c2a/spec/support/services_test_helper.rb#L1-L9

Otherwise we'll have two ways to do it, which is a bit ambiguous.